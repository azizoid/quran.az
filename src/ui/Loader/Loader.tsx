import styles from './Loader.module.scss'

export const Loader = () => (
  <div className={styles.loader}>
    <div className={styles.loadingioSpinner}>
      <div className={styles.ldio}>
        <div></div>
      </div>
    </div>
  </div>
)
