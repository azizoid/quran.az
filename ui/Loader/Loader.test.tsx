import { render } from '@testing-library/react'
import { Loader } from './Loader'

test('Render Loader', () => {
  const { container } = render(<Loader />)

  expect(container).toMatchSnapshot()
})
