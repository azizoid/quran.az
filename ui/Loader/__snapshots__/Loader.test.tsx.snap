// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Loader 1`] = `
<div>
  <div
    class="loader"
  >
    <div
      class="loadingioSpinner"
    >
      <div
        class="ldio"
      >
        <div />
      </div>
    </div>
  </div>
</div>
`;
